MobileWeather/ImageProcessor.m

```diff
@@ -31,9 +31,13 @@ - (instancetype)init {
     return self;
 }
 
-- (UIImage *)imageFromConditionImage:(NSString *)conditionImage {
-    UIImage *image = [UIImage imageNamed:conditionImage];
-    
+- (SDLArtwork *)artworkFromConditionImage:(NSString *)conditionImage imageSize:(ImageSize)imageSize {
+    return [SDLArtwork artworkWithImage:[[self imageFromConditionImage:conditionImage imageSize:imageSize] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate] asImageFormat:SDLArtworkImageFormatPNG];
+}
```

Comment:
The ordering should follow the .h file, so this should be below the `imageFromConditionImage` method